src/components/Content.js

```diff
@@ -1,62 +1,51 @@
+    closed = issues.filter(item => item.locked === false);
+    count = active.length;
+    count2 = closed.length;
 
-                <div className="ui top attached tabular menu">
-                <div className="active item">Code</div>
-                <div className=" item">Issues</div>
-                <div className=" item">Pull requests</div>
-                <div className=" item">Actions</div>
-                <div className=" item">Projects</div>
-                <div className=" item">Wiki</div>
-                <div className=" item">Security</div>
-                <div className=" item">Insights</div>
-                </div>
-                <div className="ui bottom attached active tab segment">
-               <table className="ui single line table">
-                            <thead>
-                                <tr>
-                                    <th colSpan="3">
-                                   342 Open
-                                    </th>
-                                
-                                </tr>
-                            </thead>
-                            <tbody>
-                                <tr>
-                                <td className="collapsing">
-                                    <i className="folder icon"></i> node_modules
-                                </td>
-                                <td>Initial commit</td>
-                                <td className="right aligned collapsing">10 hours ago</td>
-                                </tr>
-                                <tr>
-                                <td>
-                                    <i className="folder icon"></i> test
-                                </td>
-                                <td>Initial commit</td>
-                                <td className="right aligned">10 hours ago</td>
-                                </tr>
-                            </tbody>
-                            </table>
+
+    const renderedList = issues.map((issue, index) => {
+        return(
+           
+            <tr key={issue.id}>
+           
+            <td className="collapsing">
+                <i className="dot circle outline large green icon">
+              </i>{issue.title}
+            </td>
+            <td>{issue.user.login}</td>
+            <td className="right aligned collapsing">{issue.created_at}</td>
+            </tr>
+        ); 
+    })
+
+    return(
+        
+        <table className="ui celled striped table inverted">
+            <thead>
+                <tr>
+                    <th colSpan="3">
+                <i className="dot circle outline large white icon"></i>
+                {count} Open <i className="check small white icon"></i> {count2} Closed
+
+                <div className="ui right aligned">
+                    test
                 </div>
-       
-       </div>
-       
+                </th>
+            </tr></thead>
```

Comment:
```suggestion
            </tr>
       </thead>
```
Should be next line.